ui/src/components/App/Navigator/Navigator.scss

```diff
@@ -1,8 +1,12 @@
 .Navigator {
     .navbar {
         background-color: #CE1126;
-        a {
-            color: white;
+        a, .nav-link {
+            color: white !important;
```

Comment:
Fixing the color. Using `!important` is bad practice, but it's at least is contained within this component, so it shouldn't cause issues. I was just too lazy to figure out the selector needed to avoid using it lol